test/integration/instance/restore.test.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const chai = require('chai');
+
+const expect = chai.expect;
+const Support = require('../support');
+const { DataTypes } = require('@sequelize/core');
+const sinon = require('sinon');
+
+const current = Support.sequelize;
+
+describe(Support.getTestDialectTeaser('Instance'), () => {
+  before(function () {
+    this.clock = sinon.useFakeTimers();
+  });
+
+  afterEach(function () {
+    this.clock.reset();
+  });
+
+  after(function () {
+    this.clock.restore();
+  });
+
+  describe('restore', () => {
```

Comment:
since this test suite is only about restore, you can merge both describe into one named `Model#restore` (Instance used to be a separate class from model, now it's an instance of model)